srcpkgs/ffmpeg/template

```diff
@@ -2,7 +2,7 @@
 # audacity also needs to be bumped when a new ffmpeg version bumps libavformat's soname!
 pkgname=ffmpeg
 version=4.1.1
-revision=1
+revision=2
```

Comment:
revbump is not needed, since the resulting package should have the same content as it has currently.